openapi-v1.yaml

```
@@ -4930,6 +4963,11 @@ paths:
         description: namespace array is in (an organization name or user's username)
         type: string
         required: true
+      - name: with_metadata
+        in: query
+        description: include the metadata of the groups
```

This is for arrays, no?
```suggestion
        description: include the metadata of the arrays
```